### DISCUSSION TITLE
[BUG] issues with install jekyll  on osx

### DISCUSSION THREAD

**Discussion by Nejcc** [tabler/tabler#1072] [General]
i get errors like  when i do sudo gem install bundler jekyll. or bundle install

<img width="912" alt="image" src="https://user-images.githubusercontent.com/6236128/155856639-4495a64a-abcf-444b-84ec-7233a0a90a9b.png">

**Comment by codecalm** [comment]
@Nejcc you dont have xcode installed, try `xcode-select --install` 

https://stackoverflow.com/questions/27768420/gem-installation-error-you-have-to-install-development-tools-first

**Comment by Nejcc** [comment]
> @Nejcc you dont have xcode installed, try `xcode-select --install`
> 
> https://stackoverflow.com/questions/27768420/gem-installation-error-you-have-to-install-development-tools-first

let me try  will be nice if there is in readme for osx  install tutorial too :)

**Comment by Nejcc** [comment]
this two lines fix the problem 
```
sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install
```
